### ISSUE TITLE
Midi animation bug: botched after first repeat

### ISSUE THREAD

**Issue by ingydotnet** [paulrosen/vue-abcjs-midi-demo#1]
If you play the song in this demo, you'll see that the animation messes up after the first repeat.

**Comment by ingydotnet** [comment]
Seems that when the music reaches a repeat end marker, the animation goes back to the start of that line, rather than to the begin marker. This happens twice in the demo.

**Comment by ingydotnet** [comment]
FWIW, putting the matching repeat markers on the same line results in the animation working correctly (in this case):
```
D2|:"Em"EB{c}BA B2 EB|~B2 AB dBAG|"D"FDAD BDAD|FDAD dAFD|"Em"EBBA B2 EB|B2 AB defg|"D"afe^c dBAF|1"Em"DEFD E2 D2:|2"Em"DEFD E2 gf||
|:"Em"eB B2 efge|eB B2 gedB|"D"A2 FA DAFA|A2 FA defg|"Em"eB B2 eBgB|eB B2 defg|"D"afe^c dBAF|1"Em"DEFD E2 gf:|2"Em"DEFD E4|]
```

**Comment by paulrosen** [comment]
Thanks. This will be fixed in v3.3.3, coming soon.
